doc/Language/exceptions.pod6

```diff
@@ -174,9 +173,11 @@ to continue from the point of the exception by calling the C<.resume> method.
             when X::AdHoc { .resume }
         }
     }
+
     "OBAI".say;
-    #-> OHAI
-    #-> OBAI
+
+    # -> OHAI
+    # -> OBAI
```

Comment:
We are currently trying to establish a single comment line style, so this changes will be edited soon anyway.